tecken/api/views.py

```diff
@@ -522,6 +522,8 @@ def hydrate_upload(upload_id):
 @api_permission_required("upload.view_all_uploads")
 def upload_files(request):
     qs = _upload_files_build_qs(request)
+    if isinstance(qs, http.JsonResponse):
+        return qs
```

Comment:
`_upload_files_build_qs` can return a `JsonResponse` if the field values are invalid. This fixes handling for that case.